truffle/contracts/Splitter.sol

```diff
@@ -0,0 +1,106 @@
+pragma solidity 0.5.10;
+
+import './Owned.sol';
+
+contract Splitter is Owned {
+    
+    struct Person {
+        address wallet;
+        bytes32 name;
+        uint    pendingWithdrawals;
+        uint    index;
+        bool    isParticipant;
+    }
+    
+    mapping(address => Person) participantStorage;
+    address[] participantList;
+    uint public balance;
```

Comment:
Is this important? `address(this).balance` is the actual funds held in the contract.

Reply:
True. I missed this details.